modules/nf-scil/tracking/pfttracking/main.nf

```diff
@@ -0,0 +1,78 @@
+
+process TRACKING_PFTTRACKING {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+        tuple val(meta), path(fodf), path(seed), path(include), path(exclude)
+
+    output:
+        tuple val(meta), path("*__pft_tracking.trk")            , emit: trk
+        path "versions.yml"                                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def pft_random_seed = task.ext.pft_random_seed ? "--seed " + task.ext.pft_random_seed : ""
+    def compress = task.ext.pft_compress_streamlines ? "--compress " + task.ext.pft_compress_value : ""
+    def pft_algo = task.ext.pft_algo ? "--algo " + task.ext.pft_algo: ""
+    def pft_seeding_type = task.ext.pft_seeding ? "--"  + task.ext.pft_seeding : ""
+    def pft_nbr_seeds = task.ext.pft_nbr_seeds ? "-- "  + task.ext.pft_nbr_seeds : ""
```

Comment:
```suggestion
    def pft_nbr_seeds = task.ext.pft_nbr_seeds ? ""  +  task.ext.pft_nbr_seeds : ""
```
This provides the number of seeds for $pft_seeding_type, no? If this is the case, there shouldn't be the "-- ".